terraform/modules/postgresql-user/98-variables.tf

```
@@ -75,3 +75,9 @@ variable "redshift_schema_name_procedures" {
   type        = string
   default     = "terraform_postgresql_user_module"
 }
+
+variable "grant_groups" {
```

```suggestion
variable "grant_redshift_groups" {
```
Just to be clear that this is only for Redshift